changeset-feedback/index.ts

```diff
@@ -17,13 +17,16 @@ async function main() {
   const marker = core.getInput('marker', { required: true });
   const diffRef = core.getInput('diff-ref', { required: true });
   const issueNumberStr = core.getInput('issue-number', { required: true });
-  const botUsername = core.getInput('bot-username', {required: true});
+  const botUsername = core.getInput('bot-username', { required: true });
+  const isCommunityPluginsRepo = core.getInput('is-community-plugins-repo', {
+    required: false,
+  });
```

Comment:
Would `multiple-workspaces` or something like that cover it too do you think? Now and in the future. Just for the case of keeping it slightly more generic so that it's less awkward to potentially rely on this outside the community plugins repo if that's a thing we can easily support

Reply:
yeah ok - was a little torn on the name here, so yep, will update